commands/login.go

```diff
@@ -130,7 +130,7 @@ func RunLoginUser(c *core.CommandConfig) error {
 
 	// Store credentials
 	c.Printer.Verbose("Storing credentials to the configuration file: %v", viper.GetString(constants.ArgConfig))
-	err = config.WriteFile()
+	// err = config.WriteFile() // TODO: Replace me in a separate PR
```

Comment:
Is this already covered in another PR?

Reply:
yeah, in #353 

the line has been commented because `WriteFile` has been replaced by `Write(data)`, and I was too lazy to fix it because:
a) the target branch is not `master` - it is a holding branch `feat/config-rework`, which is not meant to be merged until the whole feature is working correctly
b) this whole login functionality was meant to be reworked anyway (by #353) - so any refactor / fixing on this current version is a waste of time

Reply:
but i can for sure fix this function call if we want full transparency here

Reply:
> but i can for sure fix this function call if we want full transparency here

no need, looks good 👍🏻  thank you for clarifying!